MANIFEST.in

```diff
@@ -0,0 +1 @@
+include setupbase.py
```

Comment:
JupyterLab and notebook both include `README.md` and `package.json` here. I don't know in which circumstances those are important, but it can't hurt.

Reply:
now added